resources/compass/charts/gateway/templates/gateway-certs-mtls.yaml

```diff
@@ -8,5 +8,4 @@ type: Opaque
 data:
   "key": {{ .Values.global.ingress.tlsKey }}
   "cert": {{ .Values.global.ingress.tlsCrt }}
-  "cacert": "" # This value is populated by Connectivity Certs Setup Job
```

Comment:
I wonder if we cannot just remove the `key` and `cert` from this secret as they are already specified in `gateway-certs.yaml`

Reply:
I don't know why there are 2 secrets for this chart (gateway-certs.yaml and gateway-certs-mtls.yaml)

Both of these use the same vars:
- {{ .Values.global.ingress.tlsKey }}
- {{ .Values.global.ingress.tlsCrt }} 

So no different values can be specified. But the connector chart refers to the mtls secret name ({{ .Values.global.connector.secrets.rootCA.name }}), and also the mtls gateway is configured to use this.  
Is this some future requirement, or can it be modified to just use one secret?

Reply:
It is recommended to use separate secrets per gateway, as in the case of a faulty secret (pending refresh, update, etc) all gateways using this secret are down. This separation allows us to keep gateways as separate entities

Reply:
But we do use the single secret in Kyma and only have `cacert` in a separate one.
Isn't it harder to then renew certs?